src/components/Editable.tsx

```diff
@@ -768,10 +768,10 @@ const Editable = ({
           isCursorCleared
           ? ''
           : isEditing
-          ? value
+          ? removeMarkdownFormatting(value)
```

Comment:
We should check `selection.isThought()` within the `isEditing case in order for this to work properly on mobile. On mobile, we only want to render the markdown when the keyboard is up.

You can simulate mobile in Chrome Developer Tools, or by hardcoding `isTouch` to true in [browser.ts](https://github.com/cybersemics/em/blob/11bd31f3a1c3b80a12a0ecdd91534c6002774c6a/src/browser.ts#L6).

Reply:
Thanks for pointing this out. I've updated the code to meet the requirements, but sometimes the markdown doesn't render even if the keyboard is up. In such case, if I tap at the thought one more time, it reveals the markdown. Can you test and guide me through the expected behavior in such cases.